test/gtest/ucp/test_ucp_wakeup.cc

```diff
@@ -117,6 +117,17 @@ UCS_TEST_P(test_ucp_wakeup, tx_wait, "ZCOPY_THRESH=10000")
     std::string send_data(COUNT, '2'), recv_data(COUNT, '1');
     void *sreq, *rreq;
 
+    if (std::find(GetParam().transports.begin(),
```

Comment:
maybe we can add common testing function like "bool has_transport(const string& tl_name)"